matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/identity/IdentityModule.kt

```diff
@@ -48,10 +49,11 @@ internal abstract class IdentityModule {
         @SessionScope
         @AuthenticatedIdentity
         fun providesOkHttpClient(@UnauthenticatedWithCertificate okHttpClient: OkHttpClient,
-                                 @AuthenticatedIdentity accessTokenProvider: AccessTokenProvider): OkHttpClient {
+                                 @AuthenticatedIdentity accessTokenProvider: AccessTokenProvider,
+                                 globalErrorReceiver: GlobalErrorReceiver): OkHttpClient {
```

Comment:
Since this is for the identity server, I do not think we can use the GlobalErrorReceiver here. I would make it nullable in `addAccessTokenInterceptor`